### PR TITLE
UI/UX design

### PR DESCRIPTION
**Pull request by Jesutoby**
_Worked on UI/UX design_

### CODE REVIEW

File: app/src/main/java/com/example/medrecord/NavigationActivity.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import androidx.core.view.GravityCompat;
+import androidx.drawerlayout.widget.DrawerLayout;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class NavigationActivity extends AppCompatActivity implements NavigationView.OnNavigationItemSelectedListener {
+    private RecyclerView mRecyclerView;
+    private WordListAdapter mAdapter;
```
Comment (olubunmialegbeleye) [inline app/src/main/java/com/example/medrecord/NavigationActivity.java:24]: This line is throwing an error: Cannot find symbol.

File: app/src/main/java/com/example/medrecord/RecordManager.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package com.example.medrecord;
```
Comment (olubunmialegbeleye) [inline app/src/main/java/com/example/medrecord/RecordManager.java:1]: This file has several errors too. Please help look into them. Thank you for the effort you are putting into this thing.

File: app/src/main/java/com/example/medrecord/NavigationActivity.java (hunk 1 above)
Comment (Jesutoby) [reply app/src/main/java/com/example/medrecord/NavigationActivity.java:24]: Resolved. Check it out now.

File: app/src/main/java/com/example/medrecord/RecordManager.java (hunk 2 above)
Comment (Jesutoby) [reply app/src/main/java/com/example/medrecord/RecordManager.java:1]: Resolved. Check it out now

**Pull request merged**